Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/LeapMotionConfigurationChecker.cs

```diff
@@ -133,9 +137,9 @@ private static void ConfigureLeapMotion(bool isLeapInProject)
         }
 
         /// <summary>
-        /// Checks if the Leap Motion Core Assets are version 4.4.0.
+        /// Checks if the Leap Motion Core Assets are version 4.4.0 or 4.5.0.
```

Comment:
Nit: Instead of specifying 4.4 or 4.5, just say "if it's a supported Leap Motion version". (Same for the returns section)

Otherwise future updates to the supported version will also require a comment change, which doesn't really scale super well.